processing/src/main/java/org/apache/druid/segment/filter/ColumnComparisonFilter.java

```diff
@@ -148,9 +148,9 @@ public boolean supportsBitmapIndex(BitmapIndexSelector selector)
   }
 
   @Override
-  public boolean shouldUseIndex(BitmapIndexSelector bitmapIndexSelector)
+  public boolean shouldUseBitmapIndex(BitmapIndexSelector selector)
   {
-    return Filters.shouldUseIndex(this, bitmapIndexSelector, filterTuning);
+    return Filters.shouldUseBitmapIndex(this, selector, filterTuning);
```

Comment:
It's confusing that this method doesn't just return `false` while `supportsBitmapIndex()` does.

Reply:
agree, changed